x11-misc/yagostatus/yagostatus-9999.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+EGO_PN="github.com/burik666/yagostatus"
+DESCRIPTION="Yet Another i3status replacement written in Go"
+HOMEPAGE="https://github.com/burik666/yagostatus"
+LICENSE="GPL-3"
+SLOT="0/${PVR}"
+
+if [[ ${PV} == *9999* ]]; then
+	inherit golang-vcs
+	EGIT_COMMIT="master"
+	EGIT_REPO_URI="https://github.com/burik666/yagostatus"
+	SRC_URI=""
+	KEYWORDS=""
+else
+	inherit golang-vcs-snapshot
+	EGIT_COMMIT="${PV}"
+	SRC_URI="https://${EGO_PN}/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+src_prepare() {
+	default
+	GOPATH="${S}:$(get_golibdir_gopath)" go get ./...
```

Comment:
is it getting from internet at this point?
this is not allowed  in `::gentoo` and simply fails to install.
Also `|| die` missing and it does not catch if it fails.


seems you are the upstream for this package, so you'll either have to fix packaging to bundle everything or use vendored deps and `EGO_VENDOR` or any other insane go packaging solution =D


```sh
>>> Emerging (1 of 1) x11-misc/yagostatus-0.2.0::gentoo
 * yagostatus-0.2.0.tar.gz BLAKE2B SHA512 size ;-) ...                                                                                                                                [ ok ]
>>> Unpacking source...
>>> Source unpacked in /tmp/portage/x11-misc/yagostatus-0.2.0/work
>>> Preparing source in /tmp/portage/x11-misc/yagostatus-0.2.0/work/yagostatus-0.2.0 ...
package golang.org/x/net/websocket: unrecognized import path "golang.org/x/net/websocket" (https fetch: Get https://golang.org/x/net/websocket?go-get=1: dial tcp: lookup golang.org: no such host)
package gopkg.in/yaml.v2: unrecognized import path "gopkg.in/yaml.v2" (https fetch: Get https://gopkg.in/yaml.v2?go-get=1: dial tcp: lookup gopkg.in: no such host)
>>> Source prepared.
>>> Configuring source in /tmp/portage/x11-misc/yagostatus-0.2.0/work/yagostatus-0.2.0 ...
>>> Source configured.
>>> Compiling source in /tmp/portage/x11-misc/yagostatus-0.2.0/work/yagostatus-0.2.0 ...
>>> Source compiled.
>>> Test phase [not enabled]: x11-misc/yagostatus-0.2.0

>>> Install yagostatus-0.2.0 into /tmp/portage/x11-misc/yagostatus-0.2.0/image/ category x11-misc
!!! dobin: bin/yagostatus does not exist
 * ERROR: x11-misc/yagostatus-0.2.0::gentoo failed (install phase):
 *   dobin failed
 * 
 * If you need support, post the output of `emerge --info '=x11-misc/yagostatus-0.2.0::gentoo'`,
 * the complete build log and the output of `emerge -pqv '=x11-misc/yagostatus-0.2.0::gentoo'`.
 * The complete build log is located at '/tmp/portage/x11-misc/yagostatus-0.2.0/temp/build.log'.
 * The ebuild environment file is located at '/tmp/portage/x11-misc/yagostatus-0.2.0/temp/environment'.
 * Working directory: '/tmp/portage/x11-misc/yagostatus-0.2.0/work/yagostatus-0.2.0'
 * S: '/tmp/portage/x11-misc/yagostatus-0.2.0/work/yagostatus-0.2.0'
 * QA Notice: file does not exist:
 * 
 *      dobin: bin/yagostatus does not exist

>>> Failed to emerge x11-misc/yagostatus-0.2.0, Log file:

>>>  '/tmp/portage/x11-misc/yagostatus-0.2.0/temp/build.log'

 * Messages for package x11-misc/yagostatus-0.2.0:

 * ERROR: x11-misc/yagostatus-0.2.0::gentoo failed (install phase):
 *   dobin failed
 * 
 * If you need support, post the output of `emerge --info '=x11-misc/yagostatus-0.2.0::gentoo'`,
 * the complete build log and the output of `emerge -pqv '=x11-misc/yagostatus-0.2.0::gentoo'`.
 * The complete build log is located at '/tmp/portage/x11-misc/yagostatus-0.2.0/temp/build.log'.
 * The ebuild environment file is located at '/tmp/portage/x11-misc/yagostatus-0.2.0/temp/environment'.
 * Working directory: '/tmp/portage/x11-misc/yagostatus-0.2.0/work/yagostatus-0.2.0'
 * S: '/tmp/portage/x11-misc/yagostatus-0.2.0/work/yagostatus-0.2.0'
```

Reply:
fixed